pymemcache/test/test_client.py

```diff
@@ -92,6 +92,15 @@ def _set():
         with pytest.raises(MemcacheIllegalInputError):
             _set()
 
+    def test_set_unicode_char_in_middle_of_key(self):
+        client = self.make_client([b'STORED\r\n'])
+
+        def _set():
+            client.set('helloworld_\xb1901520_%c3', b'value', noreply=False)
+
+        with pytest.raises(MemcacheIllegalInputError):
+            _set()
```

Comment:
Yes, I was copying. I have no good reason, do you think I should change them? I don't know if it belongs in this particular PR